openpower/overlay/etc/init.d/S99signal-linux-complete

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#
```

Comment:
@stewartsmith Is this init script supposed to be executed by every other platform as well?